prep-javascript-functions/main.js

```diff
@@ -0,0 +1,72 @@
+// addTwoNumbers //
+
+function addTwoNumbers(a, b) {
+  return a + b;
+}
+
+var addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log('addTwoNumbers Exercise:', addTwoNumbersResult);
+
+// convertHoursToMinutes //
+
+function convertHoursToMinutes(hours) {
+  return hours * 2;
+}
+
+var convertHoursToMinutesResult = convertHoursToMinutes(60);
+console.log('convertHoursToMinutes Exercise:', convertHoursToMinutesResult);
+
+// getGreeting //
+
+function getGreeting(name) {
+  return name;
+}
```

Comment:
For this one, we'll want to move the `"Hello, "` part into the function so that we can do something like:

```js
var firstGreeting = getGreeting('Josh');
var secondGreeting = getGreeting('Tim');
```

Reply:
![image](https://user-images.githubusercontent.com/31152745/204064224-b9262a04-9295-4d25-a82a-7fa90aad1ea2.png)